client/index.html

```diff
@@ -4,10 +4,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>Bus data</title>
     <link rel="stylesheet" href="https://unpkg.com/purecss@1.0.1/build/pure-min.css" integrity="sha384-oAOxQR6DkCoMliIh8yFnu25d7Eq/PHS21PClpwjOTeU2jRSq11vu66rf90/cZr47" crossorigin="anonymous">
-    <style type="text/css">
-        .container { padding: 1em 2em; max-width: 768px; margin: 0 auto; letter-spacing: normal !important; }
-        tr:hover { background-color: #f2f2f2; }
-    </style>
+    <link href="style.css" type="text/css" rel="stylesheet"></link>
```

Comment:
Nit: `<link>` does not need to be closed, and it is now best practice to omit `type="text/css"`

```suggestion
    <link href="style.css" rel="stylesheet">
```